src/item_pocket.h

```diff
@@ -72,6 +72,11 @@ class item_pocket
 
         class favorite_settings
         {
+            private:
+                // Internal function that helps extend acceptance criteria to container content.
+                // Returns true if all items inside the given container pass current whitelist and blacklist filters.
+                bool accepts_container( const item &it ) const;
```

Comment:
```suggestion
                bool accepts_container( const item &container ) const;
```

Reply:
Code refactored in https://github.com/CleverRaven/Cataclysm-DDA/pull/54812/commits/3a62df009914e6776f1c69073e1f1bedd78be382